src/en/charms-scaling.md

```diff
@@ -1,88 +1,68 @@
 Title: Scaling applications
-TODO:  The Scaling Back section should just reference charms-destroy.html (remove-unit)
-       Critical: review required
+TODO:  More direction ito how users discover "built-in scaling" (e.g. the store is not clear on wordpress)
 
 # Scaling applications
 
-One of the killer features of computing in the cloud is that it (should)
-seamlessly allow you to scale up or down your applications to meet your needs
-and whims. Juju not only makes it simple to deploy applications, but crucially
-makes it easy to manage them too. It won't anticipate you getting slashdotted
-or on the front page of hacker news (yet), but it does mean that when you do
-you can reliably scale your applications to meet the demand.
+The capability of a service to adjust its resource footprint to a level
```

Comment:
It makes me a little sad that the hyperbole in the first couple of sentences has gone, but I can see why  you've cut it for a dryer explanation.

Reply:
:disappointed: